asv_bench/benchmarks/period.py

```diff
@@ -3,84 +3,11 @@
 
 
 class PeriodProperties(object):
-    def setup(self):
-        self.per = Period('2012-06-01', freq='M')
-
-    def time_year(self):
-        self.per.year
-
-    def time_month(self):
-        self.per.month
-
-    def time_quarter(self):
-        self.per.quarter
+    params = ['M', 'min']
```

Comment:
does this have to be double brackets ?

Reply:
Not in my admittedly-limited experience.  Though the only examples that use both `params` and `param_names` do have 2+ param_names, so _do_ have double double-brackets for `params`.